metaflow/metaflow_config.py

```diff
@@ -519,8 +529,18 @@ def _new_get_pinned_conda_libs(
                     return d1
 
                 globals()[n] = _new_get_pinned_conda_libs
+            elif n == "TOGGLE_DECOSPECS":
+                if any([x.startswith("-") for x in o]):
+                    raise ValueError("Removing decospecs is not currently supported")
+                if any(" " in x for x in o):
+                    raise ValueError("Decospecs cannot contain spaces")
+                _TOGGLE_DECOSPECS.extend(o)
```

Comment:
what's the latest on this? it seems we can only add decospecs for now through extensions, not remove them.

The usage is a bit different from the PR description as well, not requiring prepending a `+` for adding a decospec. This makes sense for now as `-` is not supported, but should we agree on the final form already, or is there a use case for this yet?

Reply:
for adding decospecs the mechanism is working as expected 👍

Reply:
@savingoyal had me remove the - option since he thought it would be confusing. It can easily be added back if we feel the need for it later (I don't need it right now, I had initially done the - to keep in line with the TOGGLE_STEP_DECORATOR for example (but that one I really needed the -)).

Reply:
was more so wondering about the prefixes in general, as currently `+` is not supported either, so do we even need a check against `-`?

not working at the moment. Is this intended?
```
TOGGLE_DECOSPECS = ["+kubernetes:memory=1024"]
```

this works fine
```
TOGGLE_DECOSPECS = ["kubernetes:memory=1024"]
```
---
as its currently append-only, naming the var `toggle` is a bit confusing. We can keep it as is though if the end goal is to support toggling the decospecs.